test/sanity/validate-modules/main.py

```diff
@@ -1208,12 +1207,13 @@ def _validate_argument_spec(self, docs, spec, kwargs):
                     self.reporter.error(
                         path=self.object_path,
                         code=329,
-                        msg=('Default value from the argument_spec (%r) is not compatible '
-                             'with type %r defined in the argument_spec' % (data['default'], _type))
+                        msg=("Argument '%s' in argument_spec defines default as '%s' "
```

Comment:
Please switch back to just using `%r` here instead of `'%s'` as that assists in providing more information about the original type.

Reply:
Fixed.